src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("click on Update button.");
+ 	 		Assert.assertTrue(clickUpdateButton8.isDisplayed());
+ 	 		clickUpdateButton8.click();
+ 	 		JavaScriptExec.sleep();
+ 	 		}
+ 	 		
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--5-update-form']/div[2]/a[1]")
+ 	
+ 	 //*[@id="change--5-update-form"]/div[2]/a[1]/span
+ 	 	private WebElement clickUpdateButton12;
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void clickUpdateButton12() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("click on Update button.");
+ 	 		Assert.assertTrue(clickUpdateButton12.isDisplayed());
+ 	 		clickUpdateButton12.click();
+ 	 		JavaScriptExec.sleep();
+ 	 		}
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--9-update-form']/div[2]/a[1]")
+ 	 	//*[@id="change--9-update-form"]/div[2]/a[1]/span
+ 	 	 	private WebElement clickUpdateButton10;
+
+ 	 	 		/**
+ 	 	 		 * Method to click on Update button.
+ 	 	 		 * 
+ 	 	 		 * @throws IOException
+ 	 	 		 */
+ 	 	 	public void clickUpdateButton10() throws IOException {
+ 	 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 	 		log.info("click on Update button.");
+ 	 	 		Assert.assertTrue(clickUpdateButton10.isDisplayed());
+ 	 	 		clickUpdateButton10.click();
+ 	 	 		JavaScriptExec.sleep();
+ 	 	 		}
+ 	 	 	
+ 	 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--9-update-form']/div[2]/a[1]")
+ 	 	//*[@id="change--9-update-form"]/div[2]/a[1]/span
+ 	 	 	 	private WebElement clickUpdateButton15;
+
+ 	 	 	 		/**
+ 	 	 	 		 * Method to click on Update button.
+ 	 	 	 		 * 
+ 	 	 	 		 * @throws IOException
+ 	 	 	 		 */
+ 	 	 	 	public void clickUpdateButton15() throws IOException {
+ 	 	 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 	 	 		log.info("click on Update button.");
+ 	 	 	 		Assert.assertTrue(clickUpdateButton15.isDisplayed());
+ 	 	 	 		clickUpdateButton15.click();
+ 	 	 	 		JavaScriptExec.sleep();
+ 	 	 	 		}
+ 	 		
+ 	 	 	
+ 	 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6-update-form']/div[2]/a[1]")
+ 	 	 	
+ 	 	//*[@id="change--6-update-form"]/div[2]/a[1]/span
+ 	 	 	 	private WebElement clickUpdateButton13;
+
+ 	 	 	 		/**
+ 	 	 	 		 * Method to click on Update button.
+ 	 	 	 		 * 
+ 	 	 	 		 * @throws IOException
+ 	 	 	 		 */
+ 	 	 	 	public void clickUpdateButton13() throws IOException {
+ 	 	 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 	 	 		log.info("click on Update button.");
+ 	 	 	 		Assert.assertTrue(clickUpdateButton13.isDisplayed());
+ 	 	 	 		clickUpdateButton13.click();
+ 	 	 	 		JavaScriptExec.sleep();
+ 	 	 	 		}
+ 	 	
+ 	 	 
+ 	 	//@FindBy(how=How.XPATH, using="//*[@id="18241"]/span")
+ 	 	//*[@id="18241"]/span
+ 	 	 private WebElement selectParentOrder;
+ 	 	 	 	    /**
+ 	 	 	 	     * Method to select customer.
```

Comment:
@mayankchhajer Please remove the WebElement and format the code

Reply:
Done